api/graphql/schema.graphql

```diff
@@ -2086,6 +2086,7 @@ input CreateJobPosterTemplateInput @validator {
   description: LocalizedStringInput!
   supervisoryStatus: SupervisoryStatus! @rename(attribute: "supervisory_status")
   stream: PoolStream!
+  workStream: WorkStreamBelongsTo!
```

Comment:
Left `stream` in this one place

Reply:
This ticket was just adding `workStream` and that field was required by other stuff. The removal will happen in https://github.com/GCTC-NTGC/gc-digital-talent/issues/12142